test/module/irohad/model/converters/json_query_factory_test.cpp

```diff
@@ -127,6 +126,36 @@ TEST(QuerySerializerTest, DeserializeWhenUnknownType) {
   ASSERT_FALSE(res.has_value());
 }
 
+/**
+ * @given The json transaction that has valid and invalid hashes.
+ * @when Deserialize the json transaction.
+ * @then Validate the invalid hash is skipped and the only valid deserialized.
+ */
+TEST(QuerySerialzierTest, DeserializeGetTransactionsWithInvalidHash) {
+  JsonQueryFactory queryFactory;
+  iroha::hash256_t exact_size_hash{};
+  exact_size_hash[0] = 1;
+  QueryGenerator queryGenerator;
+  auto val =
+    queryGenerator.generateGetTransactions(0, "123", 0, {exact_size_hash});
+  val->signature = generateSignature(42);
+  auto json = queryFactory.serialize(val);
+  json.erase(std::remove_if(json.begin(), json.end(), [](auto c) {
```

Comment:
Is that necessary? And why not really clear for me?

Reply:
I added because assuming spaces in json string looks not clear. But working with not string but json value is better.